csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -25,7 +25,7 @@ public class SessionOptions:IDisposable
     {
         public IntPtr _nativePtr;
         protected static readonly Lazy<SessionOptions> _default = new Lazy<SessionOptions>(MakeSessionOptionWithCpuProvider);
-        private static string[] cudaDelayLoadedLibs = { "cublas64_100.dll", "cudnn64_7.dll" };
+        private static string[] cudaDelayLoadedLibs = { "cublas64_91.dll", "cudnn64_7.dll" };
```

Comment:
why 9.1? i thought we decided to do all future releases with cuda 10.

Reply:
@pranavsharma Did you take a look at our CMakeLists.txt ?

Reply:
We don't need to have cuda 9 pipeline any more since we decided to not support cuda 9 next release onwards.

Reply:
Reverted to Cuda10. This check for dlls  is to prevent an un-catchable exception  in case the required dlls are not available (since they dlls are delay-loaded). Need a better way to handle, which takes the versions from CMakeLists.txt instead.

Another option is to not delay-load the dlls, and let the OS do a hard check for the dlls upfront. Since there are separate CPU and GPU packages, this may not be a bad option.

Reply:
yes, there's no need to check for the dlls when the OS can do that for you.